Changes

```diff
@@ -192,6 +197,24 @@ Breaking Changes
 - TransformTool : Added private members (ABI change only - source compatibility is maintained) (#3144).
 - RenderController : Added argument to `updateInBackground()` (ABI change only, source compatibility is maintained) (#3144).
 - SceneGadget : Added new private member (ABI change only, source compatibility is maintained) (#3144).
+- LightVisualiser : The signature for `visualise()` now also includes the attributes (#3180).
+
+0.53.6.2 (relative to 0.53.6.1)
+========
+
+Improvements
+------------
+
+- CustomAttributes : Added extraAttributes plug that can be set from CompoundData (#3191)
+- StandardLightVisualiser : Setting locatorScale to 0 now avoids costly computations (#3178)
+- GafferUI : Added compatibility patch to allow >0.54 layouts to load (#3206)
+
+Fixes
+-----
+
+- MultiLineTextWidget : Fixed unicode handling in `selectedText()` (#3183)
+- SplineWidget : Fixed display transform management bug (#3187)
+- ArnoldLight : Rejig `loadShader()` to fix "nodule:type" metadata (#3188)
```

Comment:
Avoid the imperative, and make the message meaningful to the end user (nodules no longer disappear when copy/pasting Arnold quad_light and skydome_light nodes).